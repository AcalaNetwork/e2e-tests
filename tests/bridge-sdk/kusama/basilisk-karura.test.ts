import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'basilisk',
  to: 'karura',
  token: 'BSX',
})
