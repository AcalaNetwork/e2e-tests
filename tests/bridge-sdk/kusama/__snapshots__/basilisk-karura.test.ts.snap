// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'basilisk' to 'karura' using bridgeSDK cross-chain 'BSX' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 985)",
  "toChain": "(rounded 1.94)",
}
`;

exports[`'basilisk' to 'karura' using bridgeSDK cross-chain 'BSX' > fee 1`] = `"(rounded 0.06)"`;

exports[`'basilisk' to 'karura' using bridgeSDK cross-chain 'BSX' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 1000,
  "toChain": 0,
}
`;
